jupyter/README.md

```diff
@@ -338,80 +331,71 @@ The output will be:
 You have successfully simulated a quantum circuit on Google Cloud Platform using a Singularity container.
 
 
-### Running your own script {#running-your-own-script}
+### Running your own script
 
 If you want to run a Python script, you can locate a file in the home directory on your VM, then run something like in the container shell
 
-	`$ python3 /homedir/myscript.py`
+```console
+    $ python3 /homedir/myscript.py
+```
 
 
-### Exit the container {#exit-the-container}
+### Exit the container
 
 Exit the container by typing cntl-d twice. You will see the output like:
 
 
-```
+```console
     [root@79804d33f250 /]# exit
 ```
 
 
 -----
 
 
-## **Cleaning up** {#cleaning-up}
-
-_[The cleaning up section is required. Use it to ensure they don’t get billed for any extra time.]_
+## **Cleaning up**
 
 To avoid incurring charges to your Google Cloud Platform account for the resources used in this tutorial:
 
 
-### Delete the VM {#delete-the-vm}
+### Delete the VM
 
 It is usually recommended to delete the entire project you created for this tutorial, but if you want to continue using the project, you can easily either STOP the VM or Delete it. Select the VM, the either select the STOP square, or the DELETE trash can.
 
 
 
 
-![alt_text](images/image9.png "image_tooltip")
+![alt_text](images/image9.png )
 
 
-
-### Delete the project {#delete-the-project}
+### Delete the project
 
 The easiest way to eliminate billing is to delete the project you created for the tutorial.
 
 **Caution**: Deleting a project has the following effects:
 
+*   **Everything in the project is deleted.** If you used an existing project for this tutorial, when you delete it, you also delete any other work you've done in the project.
+*   **Custom project IDs are lost.** When you created this project, you might have created a custom project ID that you want to use in the future. To preserve the URLs that use the project ID, such as an **<code>appspot.com</code></strong> URL, delete selected resources inside the project instead of deleting the whole project.
```

Comment:
`**<code>appspot.com</code></strong>` probably should be one of:

`<strong><code>appspot.com</code></strong>` -> <strong><code>appspot.com</code></strong>

`**```appspot.com```**` -> **```appspot.com```**

Reply:
Went with the latter.